src/query/api/v1/handler/placement/common.go

```diff
@@ -37,18 +40,27 @@ import (
 )
 
 const (
+	// M3DBServiceName is the service name for M3DB
+	M3DBServiceName = "m3db"
+	// M3AggServiceName is the service name for M3Agg
+	M3AggServiceName = "m3agg"
+	// TODO: Delete me?
 	// DefaultServiceName is the default service ID name
-	DefaultServiceName = "m3db"
+	DefaultServiceName = M3DBServiceName
 	// DefaultServiceEnvironment is the default service ID environment
 	DefaultServiceEnvironment = "default_env"
 	// DefaultServiceZone is the default service ID zone
 	DefaultServiceZone = "embedded"
-	// HeaderClusterServiceName is the header used to specify the service name.
-	HeaderClusterServiceName = "Cluster-Service-Name"
```

Comment:
Why remove these headers?

Reply:
I'm moving the service to the URL itself. I'm gonna add back the zone/environment headers.